src/grpc_requests/aio.py

```diff
@@ -556,13 +592,27 @@ async def register_service(self, service_name):
             logger.debug(f"{service_name} registration complete")
         await super(ReflectionAsyncClient, self).register_service(service_name)
 
+    async def refresh_service_registrations(self):
```

Comment:
Is this something one would want to do asynchronously? Conceptually - what would happen if calls for pre-refresh service methods were being awaited when this call came in - presumably indicating that the state of service method could be changed.